packages/core/src/App/Components/Routes/route-with-sub-routes.jsx

```diff
@@ -15,8 +15,13 @@ import { connect } from 'Stores/connect';
 
 const RouteWithSubRoutes = route => {
     const validateRoute = pathname => {
-        if (pathname.startsWith('/cashier')) {
-            return route.path === pathname || !!(route.routes && route.routes.find(r => pathname === r.path));
+        if (pathname.startsWith('/cashier') && !pathname.startsWith('/cashier/p2p/') && !!route.routes) {
```

Comment:
can we change this to 
`if (pathname.startsWith(routes.cashier) && !pathname.startsWith(routes.cashier_p2p)` .. so on.. shouldn't that work?

Reply:
yeah, that's more cleaner I'll change this :pray: